apertium-eng-cat.eng-cat.dix

```diff
@@ -11,99 +11,116 @@
-    <sdef n="detnt" /> 
-    <sdef n="top" /> 
-    <sdef n="ant" /> 
-    <sdef n="al" /> 
-    <sdef n="cog" /> 
-    <sdef n="atn" /> 
-    <sdef n="enc" /> 
-    <sdef n="pro" /> 
-    <sdef n="tn" /> 
-    <sdef n="n" /> 
-    <sdef n="np" /> 
-    <sdef n="adj" /> 
-    <sdef n="adjant" /> 
-    <sdef n="f" /> 
-    <sdef n="m" /> 
-    <sdef n="mf" /> 
-    <sdef n="sg" /> 
-    <sdef n="pl" /> 
-    <sdef n="adv" /> 
-    <sdef n="preadv" /> 
-    <sdef n="pr" /> 
-    <sdef n="prn" /> 
-    <sdef n="rel" /> 
-    <sdef n="nn" /> 
-    <sdef n="an" /> 
-    <sdef n="aa" /> 
-    <sdef n="ind" /> 
-    <sdef n="itg" /> 
-    <sdef n="det" /> 
-    <sdef n="dem" /> 
-    <sdef n="def" /> 
-    <sdef n="cnjcoo" /> 
-    <sdef n="cnjsub" /> 
-    <sdef n="cnjadv" /> 
-    <sdef n="nt" /> 
-    <sdef n="vbser" /> 
-    <sdef n="vbhaver" /> 
-    <sdef n="vblex" /> 
-    <sdef n="vbmod" /> 
-    <sdef n="inf" /> 
-    <sdef n="pres" /> 
-    <sdef n="ger" /> 
-    <sdef n="pp" /> 
-    <sdef n="pprs" /> 
-    <sdef n="subs" /> 
-    <sdef n="p1" /> 
-    <sdef n="p2" /> 
-    <sdef n="p3" /> 
-    <sdef n="pii" /> 
-    <sdef n="ifi" /> 
-    <sdef n="fti" /> 
-    <sdef n="cni" /> 
-    <sdef n="prs" /> 
-    <sdef n="pis" /> 
-    <sdef n="imp" /> 
-    <sdef n="pos" /> 
-    <sdef n="sp" /> 
-    <sdef n="ref" /> 
-    <sdef n="sup" /> 
-    <sdef n="num" /> 
-    <sdef n="ij" /> 
-    <sdef n="sent" /> 
-    <sdef n="cm" /> 
-    <sdef n="lpar" /> 
-    <sdef n="rpar" /> 
-    <sdef n="lquest" /> 
-    <sdef n="ND" /> 
-    <sdef n="GD" /> 
-    <sdef n="web" /> 
-    <sdef n="past" /> 
-    <sdef n="subj" /> 
-    <sdef n="obj" /> 
-    <sdef n="ord" /> 
-    <sdef n="gen" /> 
-    <sdef n="guio" /> 
-    <sdef n="vbdo" /> 
-    <sdef n="sint" /> 
-    <sdef n="pred" /> 
-    <sdef n="unc" /> 
-    <sdef n="sep" /> 
+  <!--Part-of-Speech Categories-->
+    <sdef n="n" c="Noun"/> 
+    <sdef n="np"  c="Proper noun"/> 
+    <sdef n="adj"  c="Adjective"/> 
+    <sdef n="adv" c="Adverb"/> 
+    <sdef n="preadv"  c="Pre-adverb"/> 
+    <sdef n="pr" c="Preposition"/> 
+    <sdef n="prn"  c="pronoun"/> 
+    <sdef n="cnjcoo"  c="Co-ordinating conjunction"/> 
+    <sdef n="cnjsub"  c="Sub-ordinating conjunction"/> 
+    <sdef n="cnjadv"  c="Conjunctive adverb"/> 
+    <sdef n="vbser"  c="Verb 'to be'"/> 
+    <sdef n="vbhaver" c="Verb 'to have'"/> 
+    <sdef n="vblex"  c="Standard ('lexical') verb"/> 
+    <sdef n="vbmod"  c="Modal Verb"/> 
+    <sdef n="vbdo"  c="Verb 'to do'"/> 
+    <sdef n="num"  c="Numeral	"/> 
+    <sdef n="ij"  c="Interjection"/> 
+<!--Punctuation-->
+    <sdef n="sent"  c="Sentence-ending punctuation"/> 
+    <sdef n="cm"  c="Comma punctuation"/> 
+    <sdef n="lpar"  c="Left parenthesis"/> 
+    <sdef n="rpar"  c="Right parenthesis"/> 
+    <sdef n="lquest"  c="Left question/exclamation mark"/> 
+    <sdef n="guio"  c="Hyphen"/> 
+    <sdef n="apos"  c="Apostrophe"/> 
+<!--Gender-->
+    <sdef n="f" c="	Feminine"/> 
```

Comment:
shouldn't have extra space in definition

Reply:
Extra spaces deleted